src/app/transactions/transactions.component.html

```diff
@@ -1,3 +1,6 @@
+<div class="mat-app-background basic-container">
```

Comment:
where are your `basic-container` styles? If you don't have it - just remove it.
do you really need to set background for table?

Reply:
why do you add it outside of block `transactions`?